bundle/DependencyInjection/Configuration.php

```diff
@@ -73,4 +76,106 @@ private function addAsynchronousIndexingSection(ArrayNodeDefinition $nodeDefinit
                 ->end()
             ->end();
     }
+
+    private function addUsePageIndexingSection(ArrayNodeDefinition $nodeDefinition): void
+    {
+        $nodeDefinition
+            ->children()
+                ->booleanNode('use_page_indexing')
```

Comment:
Let's move this under `page_indexing`, like:

```yaml
page_indexing:
    enabled: true|false
```

Reply:
resolved in 3a5d0eff4f07d403b79eee68b2e45a2a1a23781b